smdebug/trials/trial.py

```diff
@@ -364,7 +364,44 @@ def _tensors_in_collection(self, collection) -> set:
             rval.update(self._tensors_matching_regex(regex))
         return rval
 
-    def tensor_names(self, *, step=None, mode=ModeKeys.GLOBAL, regex=None, collection=None) -> list:
+    def inputs(self, step, mode=ModeKeys.GLOBAL):
```

Comment:
Where does this function get called? Same with `labels` and `predictions` below.

Reply:
These are meant to be user facing functions.
I have added tests for them.